src/applications/vaos/components/review/TypeOfCareSection.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function TypeOfCareSection(props) {
+  if (props.data.facilityType === 'communityCare')
```

Comment:
We try to always use `{` and `}` with if statements, it avoids mistakes down the road.

Reply:
I don't understand this.

Reply:
Done